ios/PurchasesHybridCommon/PurchasesHybridCommonSwift/CommonFunctionality.swift

```diff
@@ -0,0 +1,505 @@
+//
+//  CommonFunctionality.swift
+//  PurchasesHybridCommonSwift
+//
+//  Created by Andrés Boedo on 4/19/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+import Foundation
+import StoreKit
+import Purchases
+
+
+// todo: rename back to RCCommonFunctionality
+@objc(RCCommonFunctionality) public class CommonFunctionality: NSObject {
+
+    // MARK: properties and configuration
+    @objc public static var simulatesAskToBuyInSandbox: Bool = false
+    @objc public static var appUserID: String { Purchases.shared.appUserID }
+    @objc public static var isAnonymous: Bool { Purchases.shared.isAnonymous }
+
+    @objc public static var proxyURLString: String? {
+        get { Purchases.proxyURL?.absoluteString }
+        set {
+            if let value = newValue {
+                guard let proxyURL = URL(string: value) else {
+                    fatalError("could not set the proxyURL, provided value is not a valid URL: \(value)")
+                }
+                Purchases.proxyURL = proxyURL
+            } else {
+                Purchases.proxyURL = nil
+            }
+        }
+    }
+
+    @objc public var simulatesAskToBuyInSandbox: Bool {
+        get {
+            // all other platforms already support this feature
```

Comment:
this is slightly simplified from the objc version, since we're dropping support for iOS < 11, which removes the need for some other OS checks here